microsoft/WindowsAppSDK
# Add PRI generation support for C++ libraries

Turns out that it's not true that `TargetPlatformIdentifier` is UAP for .NET version less than 5.0. I tested with a Windows Forms app with .NET v4.8. Yes, we require .NET 5 or greater for WAS, but there is no need to have a check here.

We now just set `TargetPlatformIdentifierAdjusted` to UAP for all cases. MrtCore.PriGen.targets will not be imported for projects where we don't want to use it. That's currently already the case for WAPPROJ only.

**How verified**
Commit 4 verification:
- Create a new WinUI C# app using the template.
- Go into the NuGet package. Copy paste file contents.
- Add a RESW file. Rebuild and run.
- Confirm PRI generated. Confirm correct.
